frontend/src/assets/themes.jsx

```diff
@@ -13,7 +13,7 @@ const Colors = {
     dark: '#583c87',
   },
   background: {
-    default: '#fafafa',
+    default: '#ffffff',
```

Comment:
Tu separou as categorias por espaço, mas só separou algumas, eu to tendo que olhar uma por uma pra saber se foi modificado ou não, uma boa prática é de fato só alterar no que for mexer, e commitar pequenas alterações por vez, Eu sei que n ensinei isso, to ensinando agora. Pra mim aparece um bucado de linha vermelha (que tu apagou) e verde (que tu adicionou) só que a maioria é pq tu adicionou uma espaço entre essas categorias do tema, n vai quebrar a aplicação, mas eu tive que olhar uma por uma pra saber se n tinha tido alguma alteração relevante